samples/browser/src/main/AndroidManifest.xml

```diff
@@ -84,6 +84,16 @@
         </service>
 
         <service android:name=".downloads.DownloadService" />
+        <provider
```

Comment:
Perhaps the provider should be set in the component instead? See feature-prompts for an example.

Reply:
I've tried changing this and it doesn't seem to merge properly into the sample-browser manifest.